QUARKUS-4221.md

```diff
@@ -0,0 +1,75 @@
+# QUARKUS-4221 Productization of Picocli extension
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4221
+
+PR: https://github.com/quarkus-qe/quarkus-test-suite/pull/2003
+## Scope of the testing
+### What to test
+
+The test coverage will be verified for environments: JVM for Baremetal and Openshift.
```

Comment:
Why Openshift? Will you verify any K8s specific behaviors?

Reply:
No, there is nothing specific to verify about K8s, but I thought all our tests should also be executed in Openshift as a part of our testing strategy.

Reply:
1. The guide mentions some Openshift/Kubernetes specific features. Maybe we should cover them, especially if there is no coverage in upstream.

Reply:
I've added Openshit verification in the scope of the testing, thanks.